Project.toml

```diff
@@ -19,6 +21,8 @@ Distances = "0.7, 0.8"
 FileIO = "1"
 ImageCore = "0.8.1"
 ImageInTerminal = "0.3, 0.4"
+ImageShow = "0.2"
+MosaicViews = "0.2"
```

Comment:
MosaicView is planned to be reexported by `ImageShow`

https://github.com/JuliaImages/ImageShow.jl/pull/19